backend/db_context/tables.py

```diff
@@ -42,8 +47,4 @@ class Message(ReactChatDocument):
 
 if __name__ == '__main__':
     connect('ReactChat')
```

Comment:
This is not executed normally, but you can run now this file to remove every user form the DataBase (usefull for development)